### DISCUSSION TITLE
Who is using Panda 🐼 ?

### DISCUSSION THREAD

**Discussion by segunadebayo** [chakra-ui/panda#1182] [General]
We would like to know if you're using Panda as your project.

Share your project, company, and the things you love about Panda.

Let's goo!!!! 😀

**Comment by DawidWraga** [comment]
Hi Sage,

A startup that I am part of has just completely redesigned the company website [(gumbo)](https://www.gumbo.kitchen/) and we are in the process of migrating away from chakra to use panda + ark instead.

Really great experience so far, the styling system feels very inutitive to use, it's so much easier to customise components, and I genuinely believe that this is the future of modern css. 

We decided to make the switch as the designs are very different to chakra defaults so we think it would be faster to make custom components with panda. Also we didn't want to have to migrate after implementing the new designs.

I am also currently building out my own saas startup using park ui (great preset package) and it has been a fantastic DX so far.

A massive thank you to you and all of the contributors for all of the hard work that has gone into making this package. The level of community support on discord and the pace of addressing issues on github is very impressive, it means a lot to our team and gives us lot of confidence in the future of panda.

Keep up the great work!

**Comment by DarkoKukovec** [comment]
Hi,

At @infinum, we worked on redesigning the front page of one of our projects, and we took the opportunity to use Panda there (the rest of the app uses Chakra UI) - this is already in production.
We're working on a client project that uses React Server Components, and we're using both Panda and Ark there.
Also, we're gradually refactoring an internal project and migrating from Chakra UI to the combination of RSC and Panda to gain experience with various use cases.

**Comment by andrewdoro** [comment]
Hi

At @plaace we are in the process of migrating from Stitches and styled-components to Panda. We mostly required a future proof solution for RSC, but also to align our codebase to one single styling library. We already have Panda code on production.

We are also using Radix-UI for headless UI and we found it working fine if we don't use the `styled` jsx approach because of weird props merging and TS errors, so we just use another wrapper and pass props to it with `css`.
 
What I found very cool is the different ways of writing styles from using cva, css to defineRecipe and so on. I also like the great support for tokens and Variants API (something that I really loved at stitches). 
 
The project structure is a monorepo with a Next.js app and `ui` and `preset` packages using Turborepo and deploying on Vercel. We don't have any build step for the packages, being consumed as internal-packages and transpiled by the Next app. Currently the only pain I had with Panda is the lack of documentation for our approach and remote-caching not working on Vercel (still trying to figured it out).

Really like the project so far! 🎉🎉🎉

**Reply by cschroeter** [reply]
park-ui.com 😏

**Comment by kyranjamie** [comment]
We're adopting it for the [Leather's Web Extension](https://chrome.google.com/webstore/detail/hiro-wallet/ldinpeekobnhjjdofggfgjlcehhmanlj)

**Comment by NiklasPor** [comment]
Hi we’re [LastBIM](https://lastbim.com/?consent=minimal), a german B2B startup in the contech field. We just relaunched our landing page, completely styled with PandaCSS and written in Qwik.

[Our Landing Page](https://lastbim.com/?consent=minimal)

Tech-Stack:
- Styling: PandaCSS, default preset
- Frameworks: Qwik + QwikCity
- Deployment: Vercel
- Icons: @qwikest/icons => Lucide package
- CDN (videos largely): Cloudflare Cache / R2

Development experience with Panda was pretty good, exactly what I searched for after moving from ChakraUI (our product is written with React + Chakra). As the landing page differs quite heavily we decided to go with PandaCSS instead, which also made it possible for to move to Qwik. Besides a few early bugs I can only recommend to try out / use Panda. Community support was superb and a few bugs got fixed while I was developing the site :clap: 

Also the pandacss-preset-typogrpahy helped with plain pages like our privacy page :heart: 
Thanks for the great work on Panda :rocket:

**Comment by jstcki** [comment]
I work for [Republik](https://www.republik.ch), a reader-owned digital magazine from Switzerland (German only, sorry!). We've just launched [Challenge Accepted](https://www.republik.ch/challenge-accepted), which we've built using Panda. The great thing about it was how quickly we've been able to build up a completely new design system, and that it was possible to adopt incrementally (the rest of the site still uses Glamor, which we'll be replacing with Panda going forward). My favorite feature are how powerful conditional styles are and how easy it is to [customize them](https://panda-css.com/docs/customization/conditions) – well done!

**Comment by adaboese** [comment]
I am the founder of [AIMD](https://aimd.app). Using it and loving it. I've migrated between a few frameworks, but always stuck with Panda and Panda made it very easy to migrate components between frameworks too.

**Comment by theRuslan** [comment]
Switched https://katebond.ru/en from Chakra UI to Panda CSS and the result is amazing. Next time I plan to use Panda in a larger project.

**Reply by anubra266** [reply]
Looks great

**Comment by caelinsutch** [comment]
Founder at [https://www.trycue.ai/](https://www.trycue.ai/) and we're rebuilding with Panda - was a huge Chakra fan and Panda is a much better experience then tailwind which we were using before

**Reply by NiklasPor** [reply]
Your product looks awesome, now I'm thinking on whether I should add some help steps like this to our product 😄

**Comment by omarkhatibgg** [comment]
Proudly, we are at our company Tourlane, I adopted Panda since the announcement and built our design system [Fusion You](https://fusion-you.netlify.app/) and later we start slowly to release part of our website with the new design system and panda

here some pages build with panda
https://tourlane.de/usa-reise
https://www.tourlane.de/offers/21c0c5f9-fa7a-44c3-b583-70240e77fcf9?destination=usa

**Comment by segunadebayo** [comment]
React Aria Carousel is using Panda for its documentation website here
https://react-aria-carousel.vercel.app/

**Comment by segunadebayo** [comment]
CoolContrast is an open-source web tool to analyze contrast ratios and ensure compliance with WCAG accessibility guidelines.

Built with NextJs ▲ + Panda 🐼

🌐  https://coolcontrast.vercel.app/

**Comment by segunadebayo** [comment]
TheoWilliams is a website for a London-based producer.

https://theohuewilliams.com/ 

This site is built using:

Next.13 ( Server components )
Panda CSS
Sanity CMS v3
Framer Motion

**Comment by segunadebayo** [comment]
LastBIM is a B2B startup in the contech field. It relaunched its landing page, which was completely styled with PandaCSS and written in Qwik.

https://lastbim.com/?consent=minimal (link with auto-disabled tracking)

**Comment by jjjtoikka** [comment]
Hey @segunadebayo and thanks for a great library,

I'm currently evaluating Panda to become our (Avidly) main css engine for our library (custom build, quite similar to Panda with tokens and theme) and the base theme for our future projects.

We are currently using Goober, but I would like to convert it to better SSR alternative. I really like Panda, but I'm facing some issues with it, which I need to solve (Themeprovider with custom setups, heavily used context etc, so needs a lot of rewriting).

Our Stack in the future will be `React/Gsap/Framer/Vite/Waku/Qwik/WP/Hubspot/Sanity/Builder` and then library/theme.

Cheers, Janne

**Reply by segunadebayo** [reply]
Awesome! Feel free to ask questions in the discussions. We’ll do our best to help you land your migration 👍

Getting rid of the theme provider and context will definitely help improve the runtime performance.

**Comment by lilouartz** [comment]
https://pillser.com/ is built using Panda!

**Comment by juliomuhlbauer** [comment]
I am using Panda in RELAND (https://reland.com.br).  It is a marketplace for rural real estate in Brazil. Panda is amazing!

**Comment by s-h-a-d-o-w** [comment]
I'm using panda in this [simple AI chat app](https://github.com/s-h-a-d-o-w/next-simple-chatgpt) that I created for security, cost and convenience reasons and actually have been using almost daily for months now.

**Comment by cloud-walker** [comment]
I'm using panda in our design system component library at [Translated](https://translated.com), the source code is private, and the implementation its a bit quirky, but I can share the storybook: [Babele docs](https://babele-react.vercel.app/?path=/docs/getting-started--docs).

We also recently migrate from tailwind 3, so now the library its completely written in panda, but its in a dual support phase, supporting projects that has tailwind and panda classes

**Comment by heysakamoto** [comment]
## Moto UI: 90+ accessible components built with Ark UI and Pandacss
Inspired by Hero UI and Chakra UI

<a href="https://moto-ui.app" target="_blank">
<img width="1914" height="1011" alt="Getting-Started-Moto-UI-06-14-2026_12_16_PM" src="https://github.com/user-attachments/assets/ce57929f-9a9e-498b-bb2b-55d05f71efb1" />
</a>

## Built with Moto UI
<img width="1914" height="980" alt="Folklore-06-13-2026_01_18_AM" src="https://github.com/user-attachments/assets/ad133fdf-d5db-47e3-8b34-34d256a0089c" />


[Stories](https://storybook.moto-ui.app/)
[Docs](https://moto-ui.app/docs)
